content/concept-checks.md

```diff
@@ -0,0 +1,97 @@
+
+**Solution:** Institutions can enable [Deposit Authorization](https://ripple.com/build/deposit-authorization/#deposit-authorization) on their XRP Ledger accounts by [setting the `asfDepositAuth` flag in an `AccountSet` transaction](https://ripple.com/build/transactions/#accountset-flags). This makes the account unable to accept Payment transactions. The only way for accounts with Deposit Authorization set to receive funds is through Escrow, Payment Channels, or Checks. Checks are the most straightforward, familiar, and flexible way to transfer funds if Deposit Authorization is enabled.
+
+
+## Usage
+
+Checks typically have the lifecycle described below.
+
+<!--{# Diagram sources: https://docs.google.com/drawings/d/1Ez8OZVB2TLH-b_kSFOAgfYqXlEQt4KaUBW6F3TJAv_Q/edit #}-->
+
+
+[![Check flow diagram (successful cashing)](img/checks-happy_path.png)](img/checks-happy_path.png)
+
+**Step 1:** To create a Check, the sender submits a [CheckCreate](https://ripple.com/build/transactions/#checkcreate) transaction and specifies the receiver (`Destination`), expiration date (`Expiration`), and maximum amount that may be debited from the sender's account (`SendMax`).
+
+
+**Step 2:** After the CheckCreate transaction is processed, a [Check object](https://ripple.com/build/ledger-format/#check) (`tlCheck`) is created on the XRP Ledger. This object contains the properties of the Check as defined by the transaction that created it. The object can only be modified by the sender (by canceling it with a [CheckCancel](https://ripple.com/build/transactions/#checkcancel) transaction) or receiver (by canceling it or cashing it) before the expiration time is reached. After the expiration time, anyone may cancel the Check.
+
+**Step 3:** To cash the check, the receiver submits a [CheckCash](https://ripple.com/build/transactions/#checkcash) transaction. The receiver has two options for cashing the check:
+
+* `Amount` — The receiver can use this option to specify an exact amount to cash. This may be useful for cases where the sender has padded the check to cover possible [transfer fees](https://ripple.com/build/transfer-fees/) and the receiver can only accept the exact amount on an invoice or other contract.
+
+* `DeliverMin` — The receiver can use this option to specify the minimum amount they are willing to receive from the check. If the receiver uses this option, `rippled` attempts to deliver as much as possible and will deliver at least this amount. The transaction fails if the sender doesn't have at least this amount in their account.
+
+If the sender has enough funds to cover the Check and the expiration date has not passed, the funds are debited from the sender's account and credited to the receiver's account, and the Check object is is destroyed.
```

Comment:
I'd suggest referring to only one of 'expiration time' or 'expiration date' to describe `Expiration` throughout the document: 

- Three uses of 'expiration date'
- Four uses of 'expiration time'

Reply:
Looks like the docs favor "expiration time" over "expiration date" so went with the former.

Reply:
Right. Strictly speaking, a "date" doesn't have hour/minute/second data. The XRP Ledger always uses to-the-second times, so "expiration time" is correct.